packages/sdk/wallets/wallet-walletconnect/src/wc-wallet.ts

```diff
@@ -28,7 +29,9 @@ async function waitForTruthy(getValue: () => any, attempts: number = 10) {
   throw new Error('Unable to get pairing session, did you lose internet connection?')
 }
 
-const defaultInitOptions: ClientOptions = { relayProvider: 'wss://relay.walletconnect.org' }
+const defaultInitOptions: ClientOptions = {
+  relayProvider: 'wss://walletconnect.celo-networks-dev.org',
```

Comment:
question/potential nit: can the `relayProvider` be defined as a constant and then imported in the script (see above)? Or do we expect the test script to use a separate URL at some point?

Reply:
No that's a good point. We will have a production endpoint but it's not live yet + the script won't use it.